src/Symfony/Component/Serializer/Exception/IncompleteInputDataException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+/**
+ * IncompleteInputDataException.
```

Comment:
Can you give a more explicit description or just remove this line?

Reply:
It's inspired by (almost) all other exceptions already present. Are you sure? https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Serializer/Exception/RuntimeException.php#L15